frontend/src/components/GraficoLinhasEvolucao.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <div>
+    <div class="flex justifycenter">
+      <NumeroComLegenda
+        v-if="posicaoAtual"
+        como-item
+        :numero="posicaoAtual"
+        cor="#221F43"
+        legenda="Posição atual"
+        cor-de-fundo="#e8e8e866"
+        :tamanho-do-numero="34"
+        :tamanho-da-legenda="12"
+      />
+    </div>
+    <GraficoDashboard :option="configuracaoGrafico" />
+  </div>
+</template>
+
+<script setup>
+import { computed } from 'vue';
+import Big from 'big.js';
+import GraficoDashboard from '@/components/graficos/GraficoDashboard.vue';
+import NumeroComLegenda
+  from '@/components/painelEstrategico/NumeroComLegenda.vue';
+
```

Comment:
Essa linha morta no meio das importações pode forçar reordenação e isso gera diff.

Reply:
sugestao? porque se eu coloco em linha passa dos 80 caracteres e o linter chora

Reply:
Minha sugestão é só matar a linha morta. Só isso.

Reply:
ahhhhhhhhhhhhh, eu achei que voce estava falando da quebra de linha na linha 24.  
resolvido: 3939a3a6871b6ca7c46d510aba9811ee69f85f0c